optimade/utils.py

```diff
@@ -214,7 +226,7 @@ def get_all_databases(
                     progress.print(
                         f"Retrieved databases from [bold green]{provider['id']}[/bold green]"
                     )
-            except RuntimeError as exc:
+            except Exception as exc:
```

Comment:
```suggestion
            except RunTimeError as exc:
```
This can still be RunTimeError, right? Or?